includes/reports/overview/list-table/class-sensei-reports-overview-list-table-abstract.php

```diff
@@ -137,32 +137,31 @@ public function generate_report() {
 
 		$this->csv_output = true;
 
-		// Handle orderby.
-		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No action, nonce is not required.
-		$orderby = sanitize_key( wp_unslash( $_GET['orderby'] ?? '' ) );
-		if ( empty( $orderby ) || ! array_key_exists( esc_html( $orderby ), $this->get_sortable_columns() ) ) {
-			$orderby = '';
-		}
-
-		// Handle order.
-		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No action, nonce is not required.
-		$order = sanitize_key( wp_unslash( $_GET['order'] ?? 'ASC' ) );
-		$order = ( 'ASC' === strtoupper( $order ) ) ? 'ASC' : 'DESC';
-
 		$args = array(
-			'number'  => -1,
-			'offset'  => 0,
-			'orderby' => $orderby,
-			'order'   => $order,
+			'number' => -1,
+			'offset' => 0,
 		);
 
 		// Handle search.
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No action, nonce is not required.
 		$search = sanitize_text_field( wp_unslash( $_GET['s'] ?? '' ) );
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No action, nonce is not required.
+		$orderby = sanitize_text_field( wp_unslash( $_GET['orderby'] ?? '' ) );
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No action, nonce is not required.
+		$order = sanitize_text_field( wp_unslash( $_GET['order'] ?? '' ) );
```

Comment:
Maybe we can use the new `get_order_value` method?

Reply:
[here](https://github.com/Automattic/sensei/pull/4973/files#diff-3b0db8ab59647710de5201ea0742e71848c10e6e1cd15321ef8d65fd3b736f05R142) is the change, I changed it many times while fixing tests so I'm pasting code instead of commit